### DISCUSSION TITLE
Navigation disappears after upgrading Laravel 7 to 8

### DISCUSSION THREAD

**Discussion by ronvdc** [statamic/cms#5647] [Troubleshooting] [answered]
My site is currently running Statamic 3.2 and Laravel 7. As part of an upgrade to Statamic 3.3, I'm trying to upgrade to Laravel 8 following the [instructions](https://statamic.dev/upgrade-guide/laravel-7-to-8).
Everything seemed to go as expected, but the site doesn't work ("500 Server Error"). Looking at the log file, I see "production.ERROR: Navigation \[nav_menu\] not found." I can log into Control Panel, and I can confirm that the navigation menu has disappeared.
Am I missing something here? This is a "stock" Statamic site; I haven't done any Laravel customization, etc.

Thanks!
--Ron

**Comment by ronvdc** [comment]
After manually re-creating the navigation menu, the site is working correctly with Statamic 3.2 on Laravel 8. Next step, upgrade the site to Statamic 3.3.

**Comment by ronvdc** [comment, marked as answer]
And upgrading Statamic from 3.2 to 3.3 worked without a hitch. So all is well, except that I had to re-create the navigation menu after upgrading Laravel from 7 to 8. Still don't know why that was, but ah well... It's all working now.

**Reply by jasonvarga** [reply]
Perhaps you didn't clear the cache? Anyway, glad to hear it's working.

**Reply by ronvdc** [reply]
I definitely cleared the cache, both from within Control Panel and on the file system level, but that didn't resolve the issue. Anyway, I got everything working again by manually re-creating the navigation, so all is well.
